uniffi_core/src/ffi/rustcalls.rs

```diff
@@ -42,52 +42,42 @@ use std::panic;
 #[repr(C)]
 pub struct RustCallStatus {
     pub code: RustCallStatusCode,
-    // code is signed because unsigned types are experimental in Kotlin
```

Comment:
Not completely related, but how about a comment saying that code is initialized by the foreign side to RustCallStatus::Success?

Reply:
There's already a comment few lines up that says exactly this, so I wont bother repeating it.